CHANGELOG.md

```diff
@@ -49,6 +49,8 @@ Unreleased` header.
 - On Wayland, fix `Window::set_{min,max}_inner_size` not always applied.
 - On Windows, fix inconsistent resizing behavior with multi-monitor setups when repositioning outside the event loop.
 - On Wayland, fix `WAYLAND_SOCKET` not used when detecting platform.
+- On orbital, fix `logical_key` and `text` not reported in `KeyEvent`
+- On orbital, implement `KeyEventExtModifierSupplement`
```

Comment:
Just so they are consistent with all the other entries.

```suggestion
- On Orbital, fix `logical_key` and `text` not reported in `KeyEvent`.
- On Orbital, implement `KeyEventExtModifierSupplement`.
```

Reply:
Please, implement the suggestions the way they were suggested or _consistent formatting_ or apply them directly with the github UI if you can't be bothered with all of that.